src/ember-app.js

```diff
@@ -205,10 +208,14 @@ class EmberApp {
    *                                       only
    */
   async getNewApplicationInstance() {
+    const appInstanceRetrievalStartTime = Date.now();
```

Comment:
I'd generally prefer to use something with higher precision. Mind swapping to `performance.now()` ([docs here](https://nodejs.org/dist/latest-v14.x/docs/api/perf_hooks.html#perf_hooks_performance_now)):

```js
const { performance } = require('perf_hooks');

// ...snip...
const start = performance.now();
```

Reply:
+1 to `performance.now()`, not only higher precision, more importantly it's using a monotonic clock therefore it is not subject to system clock skew or adjustments https://www.w3.org/TR/hr-time-2/#sec-monotonic-clock